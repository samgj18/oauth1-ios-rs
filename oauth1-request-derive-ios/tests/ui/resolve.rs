#[derive(oauth1_request_ios::Request)]
struct Test {
    #[oauth1(fmt = missing_fmt)]
    missing_fmt: (),

    #[oauth1(skip_if = missing_skip_if)]
    missing_skip_if: u8,
}

fn main() {}
