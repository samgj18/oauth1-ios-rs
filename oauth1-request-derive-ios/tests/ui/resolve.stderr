error[E0425]: cannot find value `missing_fmt` in this scope
 --> tests/ui/resolve.rs:3:20
  |
3 |     #[oauth1(fmt = missing_fmt)]
  |                    ^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `missing_skip_if` in this scope
 --> tests/ui/resolve.rs:6:24
  |
6 |     #[oauth1(skip_if = missing_skip_if)]
  |                        ^^^^^^^^^^^^^^^ not found in this scope
